packages/service-library/src/servicelib/client_session.py

```diff
@@ -5,26 +5,40 @@
 """
 import logging
 
-from aiohttp import ClientSession, web
+from aiohttp import ClientSession, ClientTimeout, web
 
 from .application_keys import APP_CLIENT_SESSION_KEY
+from .utils import (
+    get_http_client_request_total_timeout,
+    get_http_client_request_aiohttp_connect_timeout,
+    get_http_client_request_aiohttp_sock_connect_timeout,
+)
 
 log = logging.getLogger(__name__)
 
 
 def get_client_session(app: web.Application) -> ClientSession:
-    """ Lazy initialization of ClientSession
+    """Lazy initialization of ClientSession
 
     Ensures unique session
     """
     session = app.get(APP_CLIENT_SESSION_KEY)
     if session is None or session.closed:
-        app[APP_CLIENT_SESSION_KEY] = session = ClientSession()
+        # it is important to have fast connection handshakes
+        # also requests should be as fast as possible
+        # some services are not that fast to  reply
+        # Setting the time of a request using this client session to 5 seconds totals
+        timeout_settings = ClientTimeout(
```

Comment:
as I commented before. These should go in the ``app``. 
It is fine for now but I will probably refactor it later during the second iteration of removing "trafaret" dependency